### PR TITLE
Update POM

### PR DESCRIPTION
**Pull request by casz**
* Updated Parent POM
* Version to 2.0-snapshot
* Updated distribution repository
* Solve Upper bound dependency

### CODE REVIEW

**Comment by casz** [comment]
@jhansche Noticed you haven't done a release since back in 2013.
Updated the POM distribution repository.
Once you merge this you should be able to spin a release with: `mvn release:prepare release:perform`

**Comment by casz** [comment]
@jhansche if you'd like I can help with maintaining 👍

**Comment by casz** [comment]
http://stats.jenkins.io/plugin-installation-trend/confluence-publisher.stats.json
A few people have spun their own releases 👎

**Comment by jhansche** [comment]
@casz Please do feel free to jump in! My $dayJob hasn't had the need recently for most of the features this plugin was designed for -- that's the main reason we haven't been pushing any updates or releases.

Since it has been so long since my last release, I wonder if the `mvn release:prepare release:perform` process would even still work :)  I may need to look back through some very old emails to remember how it works.

**Comment by casz** [comment]
@jhansche sadly no write access 😐 I'll ask on IRC for a bump

**Comment by casz** [comment]
created https://github.com/jenkins-infra/repository-permissions-updater/pull/479 to add myself for now 👍

File: pom.xml [hunk 1]
```diff
@@ -113,17 +112,23 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
-    <distributionManagement>
-        <repository>
-            <id>maven.jenkins-ci.org</id>
-            <url>http://maven.jenkins-ci.org:8081/content/repositories/releases/</url>
-        </repository>
-    </distributionManagement>
+	<distributionManagement>
```
Comment (daniel-beck) [inline pom.xml]: These are unnecessary, inherited from the parent POM.

Comment (casz) [reply pom.xml]: Cheers 👍

**Pull request merged**